packages/storybook/stories/components/clipboard/clipboard.stories.tsx

```diff
@@ -0,0 +1,117 @@
+import { type Meta, type StoryObj } from '@storybook/react';
+import React from 'react';
+import { Clipboard, type ClipboardProp } from '../../../../ods-react/src/components/clipboard/src';
+import { CONTROL_CATEGORY } from '../../../src/constants/controls';
+import { excludeFromDemoControls, orderControls } from '../../../src/helpers/controls';
+
+type Story = StoryObj<ClipboardProp>;
+
+const meta: Meta<ClipboardProp> = {
+  argTypes: excludeFromDemoControls(['clearable', 'onClear', 'type', 'placeholder']),
+  component: Clipboard,
+  title: 'ODS Components/Clipboard',
+};
+
+export default meta;
+
+export const Demo: Story = {
+  argTypes: orderControls({
+    disabled: {
+      table: {
+        category: CONTROL_CATEGORY.general,
+        defaultValue: { summary: 'ø' },
```

Comment:
we can actually remove all the `defaultValue` and `type: { summary: '...' },` when control are from the main component, as it infers them correctly now